index.html

```diff
@@ -0,0 +1,273 @@
+                    </li>
+                   <li><a href="#"><img src="./images/hand-icon.png" alt="hand-icon"><a/> </li>
+                </ul>
+            </div>
+        </section>
+    </div>
+    <!-- end of header sections -->
+    
+    <!-- projects section -->
+    <div class="d-flex flex-column space-y-3 my-88 mx-24">
+        <section>
+            <div class="card d-flex flex-wrap">
+                <div>
+                    <img src="./images/project1.png" class="image-auto-h" alt="project">
+                </div>
+                <div>
+                    <h1 class="m0 font-32">Tonic</h1>
+                    <ul class="d-flex space-x-1 p-highlight">
+                        <li class="font-13">Facebook</li>
+                        <li>Full Stack</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        Experimental content creation feature 
+                        that allows users to add to an 
+                        existing story over the course of a day
+                         without spamming their friends.
+                    </p>
+                    <ul class="d-flex space-x-1 list-tags no-pad">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="btn-action" type="button">See project</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="card grid">
+                <div>
+                    <img src="./images/project2.png" class="image-auto-h" alt="project">
+                </div>
+                <div>
+                    <h1 class="m0 font-32">Multi-Post Stories</h1>
+                    <ul class="d-flex space-x-1 p-highlight">
+                        <li class="font-13">Facebook</li>
+                        <li>Full Stack</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        Experimental content creation feature 
+                        that allows users to add to an 
+                        existing story over the course of a day
+                            without spamming their friends.
+                    </p>
+                    <ul class="d-flex space-x-1 list-tags no-pad">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="btn-action" type="button">See project</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="card grid">
+                <div class="">
+                    <img src="./images/project3.png" class="image-auto-h" alt="project">
+                </div>
+                <div>
+                    <h1 class="m0 font-32">Multi-Post Stories</h1>
```

Comment:
- [ ] Make reference on lin `97` on a detailed review on why you need just one `h1` element in a project 👍